js/LPJsonPollock.js

```diff
@@ -117,6 +110,21 @@ export default class LPJsonPollock {
     return frag;
   }
 
+	validate() {
```

Comment:
you missed the json param: `validate(json: Object|string)`

Reply:
true that, fixed!